catalog/spatial/registry/registry-schema-bindings/src/test/java/org/codice/ddf/registry/schemabindings/helper/InternationalStringTypeHelperTest.java

```diff
@@ -63,6 +81,14 @@ public void testGetString() throws Exception {
     }
 
     @Test
+    public void testGetStringWithForDefaultLocale() throws Exception {
```

Comment:
Nit: Would you mind removing the `For` from the method name, i.e., `testGetStringWithDefaultLocale()`?